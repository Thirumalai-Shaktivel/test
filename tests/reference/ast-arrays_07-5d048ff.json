{
    "basename": "ast-arrays_07-5d048ff",
    "cmd": "lfortran --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/arrays_07.f90",
    "infile_hash": "43140ddb20889ad30943c7abe8f223591b5f7e6a04cfcbfaf1ad5e3e",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-arrays_07-5d048ff.stdout",
    "stdout_hash": "95dc378b9a7d7d30eebb47f2bea68e7abbaaf7464173cc3f2cc268b9",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}