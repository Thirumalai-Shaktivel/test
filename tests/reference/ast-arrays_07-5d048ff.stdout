(TranslationUnit [(Program arrays_07 [] [(ImplicitNone [])] [(Declaration (AttrType TypeInteger [] ()) [(AttrDimension [(1 10 DimensionExpr)])] [(x [] [] () None)]) (Declaration (AttrType TypeInteger [] ()) [] [(i [] [] () None)])] [(= 0 x (ArrayInitializer () [(ImpliedDoLoop [i] i 1 (FuncCallOrArray size [] [(() x ())] []) ())])) (Print 0 () [x]) (Print 0 () [(FuncCallOrArray x [] [(() () 1)] [])]) (Print 0 () [(FuncCallOrArray x [] [(3 5 1)] [])]) (Print 0 () [(FuncCallOrArray x [] [(3 () 1)] [])]) (Print 0 () [(FuncCallOrArray x [] [(() 5 1)] [])]) (Print 0 () [(FuncCallOrArray x [] [(3 5 2)] [])]) (Print 0 () [(FuncCallOrArray x [] [(() 5 2)] [])]) (Print 0 () [(FuncCallOrArray x [] [(3 () 2)] [])]) (Print 0 () [(FuncCallOrArray x [] [(3 () 2)] [])]) (Print 0 () [(FuncCallOrArray x [] [(() () 2)] [])]) (Print 0 () [(FuncCallOrArray x [] [(() () 2)] [])])] [])])
